core/modules/system/system.api.php

```diff
@@ -580,7 +587,8 @@ function hook_menu_get_item_alter(&$router_item, $path, $original_map) {
  * load and return an "abc" object with internal id 123:
  * @code
  *   function mymodule_abc_load($abc_id) {
- *     return db_query("SELECT * FROM {mymodule_abc} WHERE abc_id = :abc_id", array(':abc_id' => $abc_id))->fetchObject();
+ *     return db_query("SELECT * FROM {mymodule_abc} WHERE abc_id = :abc_id",
```

Comment:
This should either be one line, or something like:

```
 *   function mymodule_abc_load($abc_id) {
 *     return db_query(
 *       "SELECT * FROM {mymodule_abc} WHERE abc_id = :abc_id",
 *       array(':abc_id' => $abc_id)
 *     )->fetchObject();
 *   }
```

Reply:
Thanks, I remember doing that before I discovered other places with `@code` blocks (and realising they shouldn't be wrapped), but then couldn't remember where this was to go back and undo it.